oneanddone/tests/functional/conftest.py

```diff
@@ -59,6 +59,11 @@ def is_local(base_url):
     return '127.0.0.1' in base_url or 'localhost' in base_url
 
 
+@pytest.fixture(scope='session')
+def is_local2(base_url):
+    return '127.0.0.1' in base_url or 'localhost' in base_url
```

Comment:
You should be able to make the existing `is_local` session scoped, as the value will not change between tests. That way you won't need to duplicate this code.